search/indexing_api.py

```diff
@@ -223,6 +227,22 @@
     "published": {"type": "boolean"},
 }
 
+PODCAST_OBJECT_TYPE = {
+    **LEARNING_RESOURCE_TYPE,
+    "id": {"type": "long"},
+    "full_description": ENGLISH_TEXT_FIELD,
+    "url": {"type": "keyword"},
+}
+
+PODCAST_EPISODE_OBJECT_TYPE = {
```

Comment:
Do you think it would be useful to include `last_modified` here and in the serializers? It's essentially the publish date of the podcast episode.

Reply:
Good point. I was subconsciously thinking of `created_on` as the publish date but that doesn't make much sense. I'll add